doc/index.rst

```diff
@@ -1,8 +1,45 @@
-Plutus
-======
+Plutus Programming and Reference Guide
+=============================================
+
+About Plutus
+--------------------
+
+Plutus consists of Plutus Core, the programming, scripting and assembly language; 
+and Plutus Tx, the compiler that compiles the Haskell source code into Plutus Core 
+to form the on-chain part of a contract application. All of this is used in combination 
+to write Plutus scripts that run on the Cardano blockchain. Compiled Plutus Core 
+scripts or programs are able to interact with the Cardano ledger through the 
+ledger interface. 
+
+The Plutus Core Repository
```

Comment:
Unsure if we should use "Plutus" unqualified here since that is what the repository is called... maybe we should rename the repo!

Reply:
I've changed the heading to "The Plutus Core Repository"

Reply:
Wait a sec, I may have misunderstood you. Are you suggesting that perhaps this heading should be "The Plutus Repository"? I would prefer to stay with the usage you prefer about not using "Plutus" unqualified. I get that Plutus is the name of the repo at this point. But it may be useful to use the heading "The Plutus Core Repository" if that's how people should think about it conceptually. Hmmm.

Reply:
Somewhere there is a duplication of this comment that I've replied to. I'd rather go with "The Plutus Core Repository" unless that actually creates an issue. Let's chat about it when we get a chance.

Reply:
I mean "The Plutus Repository" because it's literally called `plutus`. Part of the issue is that calling it the "Plutus Core" repository isn't quite right either, since there's more than Plutus Core in there!

Reply:
OK, I've edited it to be "The Plutus Repository"